diff -up ./src/CMakeLists.txt.qml-location ./src/CMakeLists.txt
--- ./src/CMakeLists.txt.qml-location	2020-01-15 17:21:52.731120225 +0100
+++ ./src/CMakeLists.txt	2020-01-15 17:22:54.075815299 +0100
@@ -120,21 +120,21 @@ IF(IOS)
 
 	# Do not delete the comments to avoid searching for the path/filename
 
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQml/Models.2 -lmodelsplugin)
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQml/StateMachine -lqtqmlstatemachine)
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtGraphicalEffects/private -lqtgraphicaleffectsprivate)
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtGraphicalEffects -lqtgraphicaleffectsplugin)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQml/Models.2 -lmodelsplugin)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQml/StateMachine -lqtqmlstatemachine)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtGraphicalEffects/private -lqtgraphicaleffectsprivate)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtGraphicalEffects -lqtgraphicaleffectsplugin)
 
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick/Controls.2 -lqtquickcontrols2plugin)
-#	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick/Controls.2/Material -lqtquickcontrols2materialstyleplugin)
-#	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick/Controls.2/Universal -lqtquickcontrols2universalstyleplugin)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick/Controls.2 -lqtquickcontrols2plugin)
+#	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick/Controls.2/Material -lqtquickcontrols2materialstyleplugin)
+#	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick/Controls.2/Universal -lqtquickcontrols2universalstyleplugin)
 
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick/Layouts -lqquicklayoutsplugin)
-#	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick/LocalStorage -lqmllocalstorageplugin)
-#	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick/Particles.2 -lparticlesplugin)
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick/Templates.2 -lqtquicktemplates2plugin)
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick/Window.2 -lwindowplugin)
-	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/qml/QtQuick.2 -lqtquick2plugin)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick/Layouts -lqquicklayoutsplugin)
+#	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick/LocalStorage -lqmllocalstorageplugin)
+#	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick/Particles.2 -lparticlesplugin)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick/Templates.2 -lqtquicktemplates2plugin)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick/Window.2 -lwindowplugin)
+	TARGET_LINK_LIBRARIES(AusweisApp -L${QT_HOST_PREFIX}/lib/qml/QtQuick.2 -lqtquick2plugin)
 
 	SET_TARGET_PROPERTIES(AusweisApp PROPERTIES RESOURCE "${IOS_RESOURCES}")
 	SET_TARGET_PROPERTIES(AusweisApp PROPERTIES XCODE_ATTRIBUTE_INSTALL_PATH "$(LOCAL_APPS_DIR)")
